src/content/en/updates/2016/10/navigator.share.md

```diff
@@ -0,0 +1,133 @@
+
+Good news, everybody! [Matt Giuca](https://twitter.com/mgiuca){: .external } on the Chrome
+team has been working on a [simple
+API](https://github.com/WICG/web-share/blob/master/docs/interface.md){: .external } called
+[Web Share](https://github.com/WICG/web-share/blob/master/docs/explainer.md){: .external }
+that allows websites to invoke the native sharing capabilities of the host
+platform.
+
+There have been a number of ways to invoke native sharing capabilities on the
+platform, but they all have significant drawbacks.  There was [Web
+Intents](https://en.wikipedia.org/wiki/Paul_Kinlan){: .external } (dead), there
+is protocol handling (poor support on mobile), there is direct sharing to a
+well-known service URL such as Twitter's, and there is also the [Android intent:
+URL syntax](https://paul.kinlan.me/sharing-natively-on-android-from-the-web/)
+{: .external } (which was, unfortunately, Android-only, and required apps to opt-
+in).
+
+The Web Share API is important because it gives the user control of how and
+where the data is shared.
+
+In Chrome 55 (Beta as of October 2016), we've enabled an [Origin
+Trial](https://github.com/jpchase/OriginTrials/blob/gh-pages/developer-
+guide.md){: .external } that lets you integrate the Web Share API into your
+site. The origin trial means this API is not generally available to all sites;
+instead you need to register to get access during the trial phase. Over this
+time, the API will likely change and break in unexpected ways, which is why we
+are looking for as much feedback as possible.
+
+The Web Share API is a
+[promise](/web/fundamentals/getting-started/primers/promises){: .external }-based,
+ single method API that takes an object with properties named title, text and url.
+
+    navigator.share({
+        title: document.title,
+        text: window.location.href,
+        url: window.location.href
+    }).then(() => console.log('Successful share'))
+    .catch(() => console.log('Error sharing:', error));
+
+<div class="video-wrapper">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="lhUzYxCvWew"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+Once invoked it will bring up the native picker (see video) and allow you to
+share the data with the app chosen by the user.
+
+<div class="clearfix"></div>
+
+This API has a few constraints:
+
+* You need to host your site in a [secure
+  context](https://www.chromium.org/Home/chromium-security/prefer-secure-origins-for-powerful-new-features){: .external }
+  (typically https).
+* You can only invoke the API as a result of a user gesture. (For example, you can't call
+  `navigator.share()` in an onload handler.)
+* The property values that you pass into the API must all be strings.
+
+### How to get this working
+
+The process is pretty simple:
+
+1. Get the [Chrome Beta Channel on
+   Android](https://play.google.com/store/apps/details?id=com.chrome.beta&hl=en){: .external }
```

Comment:
Please remove the &hl=en bit, not all of our users use EN as their primary language.